src/components/FeatureViews/AdminPortal/UnpaidSessionCard.js

```diff
@@ -71,10 +72,20 @@ const UnpaidSessionCard = ({unpaidStudent}) => {
 		unpaidStudent.sessions_left,
 		adminUtils.calculateSessionLength(startTime, endTime)
 	);
+    
+    let studentID;
+    let courseID;
+    
+    if(!student){
+    return(
+        <Loading></Loading>
```

Comment:
change this to `<Loading small/>` to use a smaller loading image

Reply:
View in Scheduler will view the scheduler at the previously viewed time frame.  Do you want me to force it to the today view?